dist/actions/admin.d.ts

```diff
@@ -30,6 +30,7 @@ export declare function AdminActions<ActionsBase extends Constructor<TinlakePara
         setProvider: (provider: any, ethOptions?: any) => void;

```

Comment:
Is it possible to remove the `dist` folder from the repo?

Reply:
Yes, there is an open ticket for that: #171